components/Nav/Nav.tsx

```diff
@@ -11,10 +12,20 @@ interface Props {
   }[];
 }
 export function Nav({ links }: Props) {
-  const { pathname } = useRouter();
-  const isActive = (href: string) => pathname === href;
+  const { closePanel } = usePanelContext();
+  const router = useRouter();
+  const isActive = (href: string) => router.pathname === href;
   const isExternalLink = (href: string) => !href.startsWith("/");
 
+  useEffect(() => {
```

Comment:
Since we have nav links in the panel, navigation looks a lot cleaner when we close the panel on navigation.